saya menambahkan code di bagian int main ()

Arisujana opened the pull request:
selamat malam pak saya I Dewa Ketut Ari Sujana Putra saya menambhkan code di bagian int main() mohon di atensi

kyo21 reviewed (changes requested):
Ini jg code ny lgsung selesai, klo diapprove maka teman sekelompok yg lain akan dianggap tdk mengerjakan apa2

Arisujana commented:
Maaf Saaya akan ulang ngirim

Arisujana commented:
baik saya ulang dari awal

Arisujana committed:
saya menambahkan code di bagian int main ()

selamat malam pak saya I Dewa Ketut Ari Sujana Putra saya menambhkan code di bagian int main() mohon di atensi

kyo21 reviewed (changes requested):
kodenya sekarang konflik dengan yg disubmit sebelumnya oleh komang artawan. saran saya kalau begini sebaiknya di fork ulang/sync fork, kemudian clone ulang ke komputer/laptopo anda supaya dapet updatean terbaru dari komang artawan. baru ulang pull request lagi

kyo21 commented:
Skrg kodenya konflik dengan gungde jagadhita, sebaiknya di fork ulang aja yg terakhir ini (kode baru dari artawan + gungde), baru pull request lgi

Arisujana committed:
menyelesaikan dan menghakhiricode

selama malam saya I Dewa Ketut Ari Sujana Putra menyelesaikan dan menghakhiri code
